spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/query/StringBasedJdbcQuery.java

```diff
@@ -78,24 +94,76 @@ public StringBasedJdbcQuery(JdbcQueryMethod queryMethod, NamedParameterJdbcOpera
 
 	/*
 	 * (non-Javadoc)
-	 * @see org.springframework.data.repository.query.RepositoryQuery#execute(java.lang.Object[])
+	 * 
```

Comment:
Please do not reformat parts of the code that shall remain untouched. You can find our formatter settings for Eclipse (IntelliJ IDEA via Eclipse Formatter Plugin) at https://github.com/spring-projects/spring-data-build/tree/master/etc/ide.